js/interface.js

```diff
@@ -1576,7 +1576,10 @@ function validateScreenshots() {
         return size[0] === screenshot.size[0] && size[1] === screenshot.size[1];
       });
 
-      if (screenshot.appId && supportedSize) {
+      // @BUG This should be using the && logic operator. Otherwise, it's ignoring the size check.
```

Comment:
can you link this https://github.com/Fliplet/fliplet-api/pull/3246 (or the ticket you're about to open)?

Reply:
Done. Will expand on the issue description later.